### PR TITLE
System Log Changes

### PR DESCRIPTION
**Pull request by Crevox**
Please review and provide feedback

### CODE REVIEW

File: ui/00_message/ui/system_log.xml [hunk 1]
```diff
@@ -13,19 +13,19 @@
   </message>
   <message>
     <key>syslog_tail_on</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;に追従オートランしました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; auto run has been initiated.</text>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml]: This line means, "You began following &lt;NAME TARGET&gt; " it even says it in the key "tail", if you try to translate the first two kanjis on Denshi Jisho, they translate in "following"

File: ui/00_message/ui/system_log.xml [hunk 2]
```diff
@@ -13,19 +13,19 @@
...
   </message>
   <message>
     <key>syslog_tail_off</key>
-    <text xml:space="preserve">追従オートランが解除されました</text>
+    <text xml:space="preserve">Auto-run has been canceled.</text>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml]: Same for this one

File: ui/00_message/ui/system_log.xml [hunk 3]
```diff
@@ -13,19 +13,19 @@
...
   </message>
   <message>
     <key>syslog_archieve_get</key>
-    <text xml:space="preserve">アチーブメント「&lt;NAME ACHIEVE&gt;」を獲得しました</text>
+    <text xml:space="preserve">Achievement 「&lt;NAME ACHIEVE&gt;」 has been earned.</text>
   </message>
   <message>
     <key>syslog_error_tell</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;にメッセージが送れませんでした</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt;'s message could not be sent.</text>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml]: This is "Cannot send message to NAME TARGET"

File: ui/00_message/ui/system_log.xml [hunk 4]
```diff
@@ -41,99 +41,99 @@
   </message>
   <message>
     <key>syslog_friend_request</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;にフレンド申請をしました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; has been requested to be your friend.</text>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml]: I think it sound better like this "You requested friendship to NAME TARGET", or "Sent friend request to NAME TARGET" Just suggesting use the best one :)

File: ui/00_message/ui/system_log.xml [hunk 5]
```diff
@@ -41,99 +41,99 @@
...
   </message>
   <message>
     <key>syslog_friend_cancel_by_decline</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;のフレンド申請を断りました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt;'s friend request was declined.</text>
   </message>
   <message>
     <key>syslog_party_request</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;にパーティー要請しました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; has been requested to join your party.</text>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml]: This one too "Sent party invite to NAME TARGET"

File: ui/00_message/ui/system_log.xml [hunk 6]
```diff
@@ -141,43 +141,43 @@
   </message>
```
Comment (BananaFlambe) [inline ui/00_message/ui/system_log.xml:141]: This is "NAME TARGET has logged in"

File: ui/00_message/ui/system_log.xml [hunk 7]
```diff
@@ -141,43 +141,43 @@
...
   <message>
     <key>syslog_pawn_rescue_start</key>
-    <text xml:space="preserve">&lt;NAME PAWN&gt;を救出してください</text>
+    <text xml:space="preserve">&lt;NAME PAWN&gt;'s rescue mission has begun.</text>
```
Comment (SpartanVXL) [inline ui/00_message/ui/system_log.xml]: Not sure of the context here but the kana says "Please rescue (NAME PAWN)". I might be wrong.

**Comment by Crevox** [comment]
A lot of this stems from the fact I was just avoiding moving the system code to keep everything consistent (they usually put it at the beginning of the sentence). If we want to move it, that's fine, it would improve the wording.

File: ui/00_message/ui/system_log.xml [hunk 8]
```diff
@@ -393,139 +393,139 @@
   </message>
   <message>
     <key>syslog_craft_consume_item</key>
-    <text xml:space="preserve">&lt;NAME ITEM&gt;を&lt;VAL&gt;個、消費しました</text>
+    <text xml:space="preserve">&lt;NAME ITEM&gt; &lt;VAL&gt; was consumed.</text>
```
Comment (SpartanVXL) [inline ui/00_message/ui/system_log.xml]: Might want to change the order of the string, for eg. below (number of items were consumed)

<text xml:space="preserve">&lt;VAL&gt; &lt;NAME ITEM&gt;'s were consumed.</text>

**Comment by Crevox** [comment]
How about < NAME ITEM > x < VAL >?

Like Apple x 1

**Comment by BananaFlambe** [comment]
I think it's fine like you said

**Comment by Crevox** [comment]
I did like all the rest of the menus.

I feel very tired.

Please review.

I did not do the ones at the top of common_dialog because I realized that Riftcrystal did all of those down to a certain point.

**Comment by Crevox** [comment]
Tons more translations, please review, I would like to try to get Riftcrystal to push all this into a patch.

File: ui/00_message/examine_message/common/common_examine.xml [hunk 9]
```diff
@@ -63,108 +63,108 @@
...
   </message>
   <message>
-    <text xml:space="preserve">キャンセル</text>
+    <text xml:space="preserve">Cancel</text>
   </message>
   <message>
-    <text xml:space="preserve">イベント進行確認
-（全メンバーの準備完了を待ちます）</text>
+    <text xml:space="preserve">Confirm Event Progress
+(You will wait for all members to ready)</text>
```
Comment (Haplo064) [inline ui/00_message/examine_message/common/common_examine.xml]: "Wait for all party members to be ready"

File: ui/00_message/examine_message/common/common_examine.xml [hunk 10]
```diff
@@ -63,108 +63,108 @@
...
   </message>
   <message>
-    <text xml:space="preserve">準備解除（パーティー招集中止）</text>
+    <text xml:space="preserve">Unprepare (Stop gathering party)</text>
```
Comment (Haplo064) [inline ui/00_message/examine_message/common/common_examine.xml]: "Disband (Stop Gathering Party)"

File: ui/00_message/ui/ArisenCard.xml [hunk 11]
```diff
@@ -213,7 +213,7 @@
   </message>
   <message>
     <key>ARISEN_CARD_TEXT_NOJOIN</key>
-    <text xml:space="preserve">クランには所属していません。</text>
+    <text xml:space="preserve">Does not belong to a clan.</text>
```
Comment (Haplo064) [inline ui/00_message/ui/ArisenCard.xml]: Remove the "." ?

File: ui/00_message/ui/ArisenCard_win.xml [hunk 12]
```diff
@@ -213,7 +213,7 @@
   </message>
   <message>
     <key>ARISEN_CARD_TEXT_NOJOIN</key>
-    <text xml:space="preserve">クランには所属していません。</text>
+    <text xml:space="preserve">Does not belong to a clan.</text>
```
Comment (Haplo064) [inline ui/00_message/ui/ArisenCard_win.xml]: Remove the "."?

File: ui/00_message/ui/QuestReward_res.xml [hunk 13]
```diff
@@ -49,99 +49,99 @@
...
+    <text xml:space="preserve">Do you want to receive all the rewards?</text>
   </message>
   <message>
     <key>REWARD_MSG_NOTION_BUY</key>
-    <text xml:space="preserve">チェックを付けていない報酬は売却されます
-(未選択の選択報酬は破棄されます)
-よろしいですか？
-売却合計額：&lt;VAL G&gt;</text>
+    <text xml:space="preserve">Random Rewards that were not checked will be sold.
+(Select Rewards not checked will be discarded.)
```
Comment (Haplo064) [inline ui/00_message/ui/QuestReward_res.xml:85]: (Some Rewards not checked will be discarded)

File: ui/00_message/ui/QuestReward_res.xml [hunk 14]
```diff
@@ -49,99 +49,99 @@
...
+    <text xml:space="preserve">Quest Reward</text>
   </message>
   <message>
     <key>REWARD_MSG_NOTICE_SELL</key>
-    <text xml:space="preserve">チェックを付けていない報酬は売却されます
-(未選択の選択報酬は破棄されます)
-よろしいですか？
-売却合計額：&lt;VAL GOLD&gt;</text>
+    <text xml:space="preserve">Random Rewards that were not checked will be sold.
+(Select Rewards not checked will be discarded.)
```
Comment (Haplo064) [inline ui/00_message/ui/QuestReward_res.xml:142]: Some Rewards not checked will be discarded

File: ui/00_message/ui/common_dialog.xml [hunk 15]
```diff
@@ -150,218 +150,217 @@
...
   </message>
   <message>
     <key>ana_fav_friend</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;をお気に入りに登録しました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; has been registered as a favorite.</text>
   </message>
   <message>
     <key>ana_fav_friend_error</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;をお気に入りに登録できませんでした</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; could not registered as a favorite.</text>
```
Comment (Haplo064) [inline ui/00_message/ui/common_dialog.xml]: could not be registered as a favorite.

**Pull request merged**